ext/st/device.hpp.in

```diff
@@ -29,6 +29,8 @@
 #include <{{ header }}>
 %% endfor
 
+#define HRTIM_TIM_TypeDef HRTIM_TypeDef
```

Comment:
Uuuuugly hack :laughing: 

I don't get where modm generates `HRTIM_TIM_TypeDef` from.

---
```
modm/src/modm/platform/clock/../device.hpp:119:8: error: 'HRTIM_TIM_TypeDef' does not name a type; did you mean 'HRTIM_TypeDef'?
```

Reply:
Implementation of this https://modm.io/reference/module/modm-cmsis-device/#debugging-peripherals

Reply:
Yes, but whats makes modm think of `HRTIM_TIM_TypeDef` instead of `HRTIM_TypeDef` which would be correct?

Reply:
The description is wrong, the module only declares extern objects to make the debugger aware of the types. Hacky hacky.

Reply:
It regexes that out of the CMSIS Header files, so ST probably messed up again.

Reply:
This regex to be precise: https://github.com/modm-io/modm/blob/develop/ext/st/module.lb#L87

Reply:
Is this still an issue with updated cmsis-header submodule?

Reply:
Let me check that...

Reply:
Nope.